core/utils/poly-look/src/visualisations/d3-utils.js

```diff
@@ -41,3 +41,17 @@ export function wrapTexts(texts) {
     }
   });
 }
+
+/**
+ * Turns a JS Date into a d3 date
+ *
+ * @param {Date} jsDate
+ * @returns d3-Date
+ */
+export function jsDateToD3Date(jsDate) {
```

Comment:
I'm not sure I get this. How I see it, `d3.timeParse` returns a function _which returns a `Date` object_. If that's the case - what you really seem to want to do here is to remove _all_ information from the date object, except year, month and day. Is that what the function is supposed to do? If so, I'd suggest a rename. The implementation _might_ also be simplified in that case, but maybe not, JS date objects are weird.

Reply:
It's from the old lineChart code, i just took it as it did the job. Can take a deeper look at it and go for a rename

Reply:
trimTimeOfDate fine as a name?